nrf_fuel_gauge/doc/api.rst

```diff
@@ -0,0 +1,15 @@
+.. _fuel_gauge_api:
+
+API documentation
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+
```

Comment:
```suggestion
```
This ``.. contents::`` is actually not needed here, as we don't have multiple sections on the page.